dynamic_rest/serializers.py

```diff
@@ -43,10 +45,20 @@ def __init__(self, *args, **kwargs):
         super(DynamicListSerializer, self).__init__(*args, **kwargs)
         self.child.parent = self
 
+    def disable_envelope(self):
+        self.child.disable_envelope()
+
     def to_representation(self, data):
         iterable = data.all() if isinstance(data, models.Manager) else data
         return [self.child.to_representation(item) for item in iterable]
 
+    def get_natural_key(self):
+        """Get the child's natural key."""
+        return self.child.get_natural_key()
+
+    def get_url(self, pk=None):
```

Comment:
TODO(ant): change this, DRF uses the `get_XXX` convention to define method-fields. Should also consider renaming `get_model`, although that is already in master...